galaxy_ng/app/api/v3/views/sync.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.exceptions import ValidationError
+
+from django.shortcuts import get_object_or_404
+
+from pulp_ansible.app import models as pulp_models
+from pulp_ansible.app.tasks.collections import sync as collection_sync
+
+from pulpcore.plugin.tasking import enqueue_with_reservation
+from pulpcore.plugin.models import Task
+
+from galaxy_ng.app.api import base as api_base
+from galaxy_ng.app.access_control import access_policy
+from galaxy_ng.app import models
+
+
+class SyncRemoteView(api_base.APIView):
+    permission_classes = [access_policy.CollectionRemoteAccessPolicy]
+    action = 'sync'
+
+    def post(self, request: Request, *args, **kwargs) -> Response:
+        distro_path = kwargs['path']
+        distro = get_object_or_404(pulp_models.AnsibleDistribution, base_path=distro_path)
+
+        if not distro.repository or not distro.repository.remote:
+            raise ValidationError(
+                detail={'remote': f'The {distro_path} distribution does not have'
+                                  ' any remotes associated with it.'})
+
+        remote = distro.repository.remote
+
+        # check if the user is pulling from api v2 (galaxy.ansible.com)
+        # if no requirements file is set, prevent the sync from happening
+        if 'api/v2' in remote.url and not remote.requirements_file:
+            raise ValidationError(
+                detail={
+                    'requirements_file':
+                        'Syncing content from galaxy.ansible.com without specifying a'
```

Comment:
```suggestion
                        'Syncing content from galaxy.ansible.com without specifying a '
```

missing a whitespace at the end